recipes/pdftools/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.37" %}
+{% set sha256 = "634cb50bcd276dda5ab39e3602acfb0370694b61b70eb357201f2bd40240ab4e" %}
+
+package:
+  name: pdftools
```

Comment:
I am not sure that we can fully avoid name conflicts.  PyPI takes the stance that the name is claimed by whoever got their first.  Out of shear practicality, maybe that is the best we can do.